packages/functions/src/definitions.ts

```diff
@@ -54,28 +59,28 @@ export interface CallByUrlOptions extends CallOptions {
 /**
  * @since 6.1.0
  */
-export type CallByUrlResult = CallResult;
+export type CallByUrlResult<ResponseData = unknown> = CallResult<ResponseData>;
 
 /**
  * @since 6.1.0
  */
-export interface CallOptions {
+export interface CallOptions<RequestData = unknown> {
   /**
    * The data to pass to the callable function.
    *
    * @since 6.1.0
    */
-  data?: any;
+  data?: RequestData | null;
```

Comment:
```suggestion
  data?: RequestData;
```

There is no reason for `null`, is there? If you don't need the param, you simply don't set it.

Reply:
I aligned the types to the js SDK:
```
export declare type HttpsCallable<RequestData = unknown, ResponseData = unknown> = (data?: RequestData | null) => Promise<HttpsCallableResult<ResponseData>>;
```
where the data can be null.

But you can tell me and I use `data?: RequestData;` instead.

Reply:
Okay, in that case, we'll leave it like this for now.